tasks/components/layout/taskDetailView/TaskDetailViewSidebar.tsx

```diff
@@ -0,0 +1,182 @@
+import { type PropsForTranslation, useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { useState } from 'react'
+import { useAuth } from '@helpwave/api-services/authentication/useAuth'
+import {
+  useAssignTaskMutation,
+  useTaskUpdateMutation,
+  useUnassignTaskMutation
+} from '@helpwave/api-services/mutations/tasks/task_mutations'
+import type { TaskDTO } from '@helpwave/api-services/types/tasks/task'
+import { tw } from '@helpwave/common/twind'
+import { ConfirmDialog } from '@helpwave/common/components/modals/ConfirmDialog'
+import { Span } from '@helpwave/common/components/Span'
+import { Button } from '@helpwave/common/components/Button'
+import { X } from 'lucide-react'
+import { Input } from '@helpwave/common/components/user-input/Input'
+import { formatDate } from '@helpwave/common/util/date'
+import { TimeDisplay } from '@helpwave/common/components/TimeDisplay'
+import type { TaskDetailViewTranslation } from './defaultTaskDetailViewTranslation'
+import { defaultTaskDetailViewTranslation } from './defaultTaskDetailViewTranslation'
+import { AssigneeSelect } from '@/components/selects/AssigneeSelect'
+import { TaskStatusSelect } from '@/components/selects/TaskStatusSelect'
+import { TaskVisibilitySelect } from '@/components/selects/TaskVisibilitySelect'
+
+type Props = {
```

Comment:
If a component is exported, its Prop Type should be too, to allow for easy extensibility.
```suggestion
export type TaskDetailViewSidebarProps = {
```